doc/tutorials/launch_uos.sh

```diff
@@ -0,0 +1,452 @@
+#!/bin/bash
```

Comment:
Is this a replacement for the launch_uos.sh script we currently ship with?

Reply:
This script for launching UOS on UP2 is modified from the script for apl-mrb, and some changes:
1. Modify "/dev/mmcblk1" -> "/dev/mmcblk0";
2. Modify "mem_size" to "512M";
3. Dual core on UP2 board;
4. Delete SD card passthrough;